### PR TITLE
feat: add library album apis

### PR DESCRIPTION
**Pull request by conradludgate**
Similar to how you can save tracks to your library, you can do the same with albums.

I copied the code directly from the tracks apis. It could probably be refactored to shared code a bit more

### CODE REVIEW

**Comment by strideynet** [comment]
Thanks for this PR, I'll take a look at this tomorrow and test it

File: library_test.go [hunk 1]
```diff
@@ -55,3 +55,54 @@ func TestRemoveTracksFromLibrary(t *testing.T) {
...
+
+func TestRemoveAlbumsFromLibrary(t *testing.T) {
+	client, server := testClientString(http.StatusOK, "")
+	defer server.Close()
+
+	err := client.RemoveAlbumsFromLibrary("4iV5W9uYEdYUVa79Axb7Rh", "1301WleyT98MSxVHPZCA6M")
+	if err != nil {
+		t.Error(err)
+	}
+}
```
Comment (strideynet) [inline library_test.go:108]: Just be picky, can I get a newline at the end of this file please

**Review by strideynet — CHANGES_REQUESTED** [review]
PR looks good, just need a newline :)

File: library_test.go (hunk 1 above)
Comment (conradludgate) [reply library_test.go:108]: Sure. It usually does it automatically but I was testing out codespaces :sweat_smile:

**Pull request merged**